src/integrationTest/groovy/wooga/gradle/unity/tasks/ReturnLicenseTaskIntegrationTest.groovy

```diff
@@ -0,0 +1,99 @@
+package wooga.gradle.unity.tasks
+
+import wooga.gradle.unity.UnityPluginTestOptions
+import wooga.gradle.unity.UnityTaskIntegrationTest
+
+class ReturnLicenseTaskIntegrationTest extends UnityTaskIntegrationTest<ReturnLicense> {
+
```

Comment:
I needed to add these lines since I don't have an active Unity license. So we must trick the system into thinking we have one.

```groovy
    def setup() {
        //setup fake license dir so we don't delete actual licenses
        def licenseDir = File.createTempDir("unity","testLicenseDir")
        createFile("testLicense", licenseDir)
        buildFile << """
        unity {
            licenseDirectory.set(new File("${escapedPath(licenseDir.path)}"))
        }
        """.stripIndent()
    }
```

Reply:
Done.